protos/feast/core/DataSource.proto

```diff
@@ -76,9 +104,9 @@ message DataSource {
     // Kafka topic to collect feature data from.
     string topic = 2;
 
-    // Classpath to the generated Java Protobuf class that can be used to decode
-    // Feature data from the obtained Kafka message
-    string class_path = 3; 
+    // Defines the data format encoding feature/entity data in Kafka messages.
+    // Kafka Data Sources support Avro and Proto as data formats.
+    DataFormat message_format = 3;
```

Comment:
So now we can should expect parquet from Kafka?
Sounds a little bit strange

Reply:
Maybe we should separate batch formats / streaming formats?

Reply:
Add `DataSourceValidator` to make  sure bad data source/data format combinations are rejected by Core  on apply.